### ISSUE TITLE
Format failed on Windows 10

### ISSUE THREAD

**Issue by zhea55** [marekpiechut/sublime-rome-formatter#2]
```bash
Traceback (most recent call last):
  File "./python3.3/subprocess.py", line 1104, in _execute_child
OSError: [WinError 193] %1 is not a valid Win32 application

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\xx\scoop\apps\sublime-text\current\Lib\python33\sublime_plugin.py", line 1494, in run_
    return self.run(edit)
  File "C:\Users\xx\scoop\persist\sublime-text\Data\Installed Packages\Rome Formatter.sublime-package\RomeFormatter.py", line 109, in run
  File "C:\Users\xx\scoop\persist\sublime-text\Data\Installed Packages\Rome Formatter.sublime-package\RomeFormatter.py", line 88, in format_code
  File "./python3.3/subprocess.py", line 819, in __init__
  File "./python3.3/subprocess.py", line 1110, in _execute_child
OSError: [WinError 193] %1 is not a valid Win32 application
```

I've tried reinstalling python through scoop, still not working. 

Please help.

**Comment by marekpiechut** [comment]
Hi,
Could you check if you have Rome formatter installed via NPM or available in path?
(that you can execute Rome from command prompt or if it't in node_modules/.bin/ folder)

If it's there please let me know, I'll try to find a Windows machine and test it out.

**Comment by zhea55** [comment]
> Hi, Could you check if you have Rome formatter installed via NPM or available in path? (that you can execute Rome from command prompt or if it't in node_modules/.bin/ folder)
> 
> If it's there please let me know, I'll try to find a Windows machine and test it out.

I've already installed.

![image](https://user-images.githubusercontent.com/7816215/214193400-bf9134f6-79ec-4e85-a953-5a922baf788e.png)

**Comment by marekpiechut** [comment]
Hi. Just wanted to let you know that I've found a Windows VM and can reproduce the issue.

**Comment by marekpiechut** [comment]
It should be fixed in version 1.0.4. Should show up in Package Control later today.
